damage_pyro_volcanyro.sp

```diff
@@ -192,7 +192,7 @@ stock GiveGiantPyro(client)
 
 		CreateRoboHat(client, FeatheredFiend, 10, 6, 1315860, 1.0, -1.0); 
 		CreateRoboHat(client, DeitysDress, 10, 6, 1315860, 1.2, -1.0); 
-		CreateRoboHat(client, PyromancersMask, 10, 6, 1315860, 0.75, -1.0); 
+		CreateRoboHat(client, PyromancersMask, 10, 6, 1315860, 1.2, -1.0); 
```

Comment:
1. scale of `1.2` seems a bit big. last time we agreed on `1.0`
2. paints are still not applied, since they are integers instead of the expected floats (ie. needs `1315860.0` instead of `1315860`)
3. speed is still reduced to 75% from last PR, might want to change it to speed him up again?